File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.py
def solution(num_list):
    result = []
    for i in num_list:
        add = 0
        while (i != 1):
            if (i % 2 == 0):
                i = i // 2
                add += 1
            elif (i % 2 != 0):
                i = (i - 1) // 2
                add += 1
        result.append(add)
    return sum(result)

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.js
function solution(num_list) {
    let result = 0;
    for (let i=0; i<num_list.length; i++) {
        while (num_list[i] != 1) {
            num_list[i] = Math.floor(num_list[i] / 2);
            result ++;
        }
    }
    return result;
}